.travis.yml

```diff
@@ -110,6 +108,8 @@ matrix:
     env: SUITE=client RDBMS=sqlite RUCIO_HOME=`pwd`
   - python: 3.5
     env:  SUITE=all RDBMS=postgres
+  - python: 3.6
+    env:  SUITE=all RDBMS=postgres
```

Comment:
i don't understand why this was moved?

Reply:
I added the py3.6 tests as discussed with @bari12

Reply:
@mlassnig The test was in the `exclude` section before, now it is in `allow_failures`